test/gpu/native/arrays/arraysInKernel.chpl

```diff
@@ -0,0 +1,12 @@
+use GpuDiagnostics;
+
+on here.gpus[0] {
+    startGpuDiagnostics();
+    @assertOnGpu
+    foreach 0..<20 {
+        var a : [1..10] int;
+    }
+    stopGpuDiagnostics();
+    assertGpuDiags(kernel_launch_aod=1, kernel_launch_um=1, host_to_device=0,
```

Comment:
just use the `kernel_launch` argument if the numbers are the same for aod and um